docker/chronos-nightly/Dockerfile

```diff
@@ -20,6 +20,16 @@ WORKDIR /opt/work
 ARG DEBIAN_FRONTEND=noninteractive
 ENV TZ=Asia/Shanghai
 
+# model: select from pytorch, tensorflow, prophet, arima, ml
+ARG model
+# auto_tuning: y (for yes) or n (for no)
+ARG auto_tuning
+# hardware: single or cluster
+ARG hardware
+# extra_dep: install extra dependencies(e.g. onnxruntime, jupyter, ...)
+#            y (for yes) or n (for no)
+ARG extra_dep 
```

Comment:
should we just give these parameters a default value? (pytorch|single|no tuning|no extra_dep)

Reply:
done.